client/src/components/SingleSelectField/SingleSelectField.js

```diff
@@ -11,57 +11,6 @@ class SingleSelectField extends Component {
     this.handleChange = this.handleChange.bind(this);
   }
 
-  /**
-   * Builds the select field in readonly mode with current props
-   *
-   * @returns {object}
-   */
-  getReadonlyField() {
-    let label = this.props.source
-      && this.props.source.find((item) => item.value === this.props.value);
-
-    label = typeof label === 'string'
-      ? label
-      : (this.props.value || '');
-
-    return <Input plaintext {...this.getInputProps()}>{label}</Input>;
```

Comment:
This seems very unnecessary

I've fixed it in a much simpler way to match the existing functionality here https://github.com/silverstripe/silverstripe-admin/pull/1128/files#diff-a5a378b351641dae0c882471702eedbfR27